.github/workflows/docker-build.yml

```diff
@@ -17,9 +17,29 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v2
 
-    - id: get-repo
-      name: Get Docker Image Identifiers
-      run: echo "::set-output name=IMAGE_REPOSITORY::$(echo ${{ github.repository }} | tr '[:upper:]' '[:lower:]')"
+    - name: Prepare
+      id: prep
+      run: |
+        DOCKER_IMAGE=${{ env.REGISTRY }}/${{ github.actor }}/${GITHUB_REPOSITORY#*/}
```

Comment:
For docker repos the name needs to be in lower case, but my username is in upper case, please use something like the code that's removed to use a lower case name.

Reply:
Fixed in newest commit, thanks for spotting this.